rg_instructor_analytics/static/rg_instructor_analytics/js/instructor_analytics.js

```diff
@@ -28,22 +191,23 @@
         var $content;
```

Comment:
I would set variable value in the definition line.

Reply:
Done